docs/scully-cmd-line.md

```diff
@@ -133,6 +134,20 @@ npx scully serve/watch --ssl --ssl-key=./url/to/file
 
 Adds a url to an ssl key file for a server with SSL.
 
+## highlight
+
+```bash
+npx scully serve/watch --hl
+```
+
+Add highlight.js render into the markdown's files.
```

Comment:
Change
from : `Add highlight.js render into the markdown's files.`
to : `Add highlight.js to render into the markdown's files.`